src/components/Posts/ComposeReply.tsx

```diff
@@ -112,9 +132,30 @@ const ComposeReply = ({
               <IconButton size="small">
                 <AddPhotoAlternateOutlinedIcon />
               </IconButton>
-              <IconButton size="small">
+              <IconButton
+                size="small"
+                onClick={toggleEmojiPicker}
+                id="emoji-button"
+              >
                 <EmojiEmotionsOutlinedIcon />
               </IconButton>
+              {showEmojiPicker && (
+                <Box
+                  sx={styles.emojiContainer}
+                  ref={emojiContainerRef}
+                  tabIndex={0}
+                  onBlur={clickOffEmojiPicker}
+                >
+                  <EmojiPicker
```

Comment:
After adjusting the paper props for the dialog component within `RepliesModal.tsx`, I think we should adjust the style of the picker. 

Here is what I recommend: 


```suggestion
                  <EmojiPicker
                    style={styles.emojiPicker as React.CSSProperties}
                    emojiStyle={EmojiStyle.TWITTER}
                    onEmojiClick={(emoji) => {
                      setPostTextContent(
                        (prevContent) => prevContent + emoji.emoji
                      );
                    }}
                    previewConfig={{ showPreview: false }}
                  />

```

look at styling comment for style adjustments.

Reply:
Implemented your suggested changes for this. I did not include the as React.CSSProperties part.